.github/CONTRIBUTING.md

```diff
@@ -1,41 +1,40 @@
 # Contributing to DeepChem
 
-We actively encourage community contributions to DeepChem. The first place to start getting involved is by running our examples locally. Afterwards, we encourage contributors to give a shot to improving our documentation. While we take effort to provide good docs, there's plenty of room for improvement. All docs are hosted on Github, either in `README.md` file, or in the `docs/` directory.
+We actively encourage community contributions to DeepChem. The first place to start getting involved is [the tutorials](https://deepchem.readthedocs.io/en/latest/get_started/tutorials.html). Afterwards, we encourage contributors to give a shot to improving our documentation. While we take effort to provide good docs, there's plenty of room for improvement. All docs are hosted on Github, either in `README.md` file, or in the `docs/` directory.
 
 Once you've got a sense of how the package works, we encourage the use of Github issues to discuss more complex changes, raise requests for new features or propose changes to the global architecture of DeepChem. Once consensus is reached on the issue, please submit a PR with proposed modifications. All contributed code to DeepChem will be reviewed by a member of the DeepChem team, so please make sure your code style and documentation style match our guidelines!
 
-
 ## Pull Request Process
+
 Every contribution, must be a pull request and must have adequate time for review by other committers.
 
 A member of the Technical Steering Committee will review the pull request. The default path of every contribution should be to merge. The discussion, review, and merge process should be designed as corrections that move the contribution into the path to merge. Once there are no more corrections, (dissent) changes should merge without further process.
 
 On successful merge the author will be added as a member of the DeepChem organization.
 
-## Code Style Guidelines
-DeepChem uses [yapf](https://github.com/google/yapf) to autoformat code.
+## Coding Conventions
 
-``` bash
-pip install yapf==0.22.0
-cd <git_root>
-yapf -i <python_files changed>
-```
+DeepChem uses these tools or styles for keeping our codes healthy.
 
-Our integration tests will fail if code is not formatted correctly
+- [YAPF](https://github.com/google/yapf) (code format)
+- [Flake8](https://flake8.pycqa.org/en/latest/) (code style check)
+- [mypy](http://mypy-lang.org/) (type check)
+- [Numpy Doctsring](https://numpydoc.readthedocs.io/en/latest/index.html) (API documents)
+- [doctest](https://docs.python.org/3/library/doctest.html) (interactive examples)
 
-## Documentation Style Guidelines
```

Comment:
Should we keep this text around about numpy documentation style here or is it already well covered in the coding conventions page?